### PR TITLE
Reformat Code Style

### PR DESCRIPTION
**Pull request by amattu2**
### Overview

This PR restyles our codebase using Prettier and Eslint.

TODO:
- [x] need to verify CustomTick functionality did not change
- [x] verify line breaks did not get messed up in APITokenDialog (visually)
- [ ] needs a comprehensive review and testing to make sure nothing changed (visually or otherwise) 
  - [x] Submission Request (A, B, C, D, Review)
  - [x] Submission Request Dialogs (History, Comments, etc?)
  - [x] Data Submission (List, Dashboard, Tabs)
  - [x] Data Submission Dialogs (History, Comments, "Delete" metadata, etc)
  - [x] User Profile
  - [x] Edit/Manage User
  - [x] User List
  - [x] Edit/Manage Organization
  - [x] Organization List
  - [x] Homepage (Logged in/Logged out)

### Change Details (Specifics)

- Fix linting errors and warnings (see commits for specifics)
- Increase Prettier `printWidth` to 100 characters
- Install missing dependencies:
  - react-transition-group
  - graphql-tag

### Related Ticket(s)

N/A

### CODE REVIEW

File: src/content/status/Page404.tsx
Comment (amattu2) [inline src/content/status/Page404.tsx:1]: This page is never even used, nor implemented. Eventually need to address that.

File: src/router.tsx [hunk 1]
```diff
@@ -37,68 +40,104 @@ const RequireAuth: FC<RequireAuthProps> = ({ component, redirectPath, redirectNa
...
+            redirectPath="/organizations"
+            redirectName="Organization Management"
+          />
+        ),
       },
       {
-        path: '*',
-        element: <Status404 />
+        path: "*",
+        element: <Status404 />,
```
Comment (amattu2) [inline src/router.tsx:137]: This route is never reached due to wildcard `:redirect` on homepage.

**Review by amattu2 — COMMENTED** [review]
Did a line-by-line skim of all changed files, with the exception of:

- Test files (*.test.ts[x])
- GraphQL queries (`src`/`graphql`/`*`)

Everything looks good to me.

I also annotated some things I noticed that are unrelated to this PR. Some of which may need action in the future.

**Comment by Alejandro-Vega** [comment]
> Did a line-by-line skim of all changed files, with the exception of:
> 
> * Test files (*.test.ts[x])
> * GraphQL queries (`src`/`graphql`/`*`)
> 
> Everything looks good to me.
> 
> I also annotated some things I noticed that are unrelated to this PR. Some of which may need action in the future.

Good catches. I agree we can address these in a separate PR.

**Review by Alejandro-Vega — APPROVED** [review]
LGTM! If we find any additional changes need to be made after this PR, we can always make updates

**Pull request merged**